torch/_functorch/vmap.py

```diff
@@ -149,14 +149,48 @@ def _process_batched_inputs(
 
 # Creates BatchedTensors for every Tensor in arg that should be batched.
 # Returns the (potentially) batched arguments and the batch_size.
+def _add_batch_dim_in_graph(self, batch_dim, level):
+    """
+    Thin wrapper around torch._C._add_batch_dim that is used to proxy in
+    PT2 export/compile fx graph
+    """
```

Comment:
description should mention non-strict tracing. this all isn't needed for dynamo tracing